src/main/resources/templates/java/gradle_gradle/readme

```diff
@@ -0,0 +1,102 @@
+# Sorting with the Strategy Pattern
+
+In this exercise, we want to implement sorting algorithms and choose them based on runtime specific variables.
+
+**Note:** This project is using `Gradle`! You have to import the project as Gradle project (not as Eclipse project) as otherwise, errors will occur and you won't be able to work on this exercise.
+
+### Part 1: Sorting
+
+First, we need to implement two sorting algorithms, in this case `MergeSort` and `BubbleSort`.
+
+**You have the following tasks:**
+
+1. [task][Implement Bubble Sort](testBubbleSort())
```

Comment:
Optional change:
Instead of declaring them here, you could also filter out the empty brackets on the server-side. This would keep the test names consistent with Maven and make porting existing Maven to Gradle exercises easier.
But it is also fine to do it how you do it now 🙂

Reply:
We could do that. But in my opinion, this would cause the code to be more complex for this one project type. And given the fact, that porting exercises is a rare use case and that changing the test names manually can easily be done, I would refrain from doing this.